### PR TITLE
[13.0][IMP]pos_default_partner: Show default partner

### PR DESCRIPTION
**Pull request by manuelregidor**
Show default partner on POS screen

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

File: pos_default_partner/static/src/js/models.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+odoo.define("pos_default_partner_show.pos_default_partner_show", function(require) {
```
Comment (ivantodorovich) [inline pos_default_partner/static/src/js/models.js]:
```suggestion
odoo.define("pos_default_partner.models", function(require) {
```

File: pos_default_partner/static/src/js/models.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            var return_val = _super_order.get_client.apply(this, arguments);
+            if (return_val == undefined) {
+                return_val = this.pos.db.get_partner_by_id(
+                    this.pos.config.default_partner_id[0]
+                );
+            }
+            return return_val;
+        },
+    });
+});
```
Comment (ivantodorovich) [inline pos_default_partner/static/src/js/models.js:19]:
```suggestion
    return models;
});
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor comments, otherwise LGTM 👍
Good improvement ☺️

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**